controllers/disruption_controller.go

```diff
@@ -294,12 +294,24 @@ func (r *DisruptionReconciler) startInjection(instance *chaosv1beta1.Disruption)
 			}
 
 			targetNodeName = pod.Spec.NodeName
-
 			// get IDs of targeted containers or all containers
 			containerIDs, err = getContainerIDs(&pod, instance.Spec.Containers)
 			if err != nil {
 				return fmt.Errorf("error getting target pod container ID: %w", err)
 			}
+
+			// filter containers
+			var removed []string
```

Comment:
Could you add an e2e-test for this new feature? it should be possible to work it into an existing one, and just change the number of expected injectors?